mizu/kernel/build.rs

```diff
@@ -5,7 +5,10 @@ fn main() {
         let manifest_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
         let out_dir = env::var("OUT_DIR").unwrap();
         let ldscript = fs::read_to_string(PathBuf::from(manifest_dir).join("link.ld")).unwrap();
+
         let new = ldscript.replace("KERNEL_START", &config::KERNEL_START.to_string());
+        let new = new.replace("MAX_HARTS", &config::MAX_HARTS.to_string());
```

Comment:
Moved `MAX_HARTS` into `config`, instead of hardcoded in the linker script file.